golang/tracereporter.go

```diff
@@ -64,7 +64,7 @@ func NewBinaryAnnotation(key string, value interface{}) BinaryAnnotation {
 
 // TraceReporter is the interface used to report Trace spans.
 type TraceReporter interface {
-	Report(span Span, annotations []Annotation, binaryAnnotations []BinaryAnnotation)
+	Report(span Span, annotations []Annotation, binaryAnnotations []BinaryAnnotation) (bool, *error)
```

Comment:
Actually `*error` is a weird thing to return - pretty much all go code returns `error`.